### ISSUE TITLE
Sentry/Sentry.h file not found

### ISSUE THREAD

**Issue by ChrisColeAttensi** [getsentry/sentry-unity#465]
### Environment

How do you use Sentry?
Sentry SaaS (sentry.io)

Which SDK and version?
Unity 0.8.0

### Steps to Reproduce

1. Build for iOS on windows PC
2. zip and copy to mac
3. unzip and open in xcode
4. archive

### Expected Result

Build not failing

### Actual Result

Build fails with an error: 'Sentry/Sentry.h' file not found.

**Comment by bitsandfoxes** [comment]
Hey @ChrisColeAttensi, sorry to see you running into this issue. Could you take a look if you locate Sentry at `Frameworks/Sentry.framework`? Just to narrow it down?

**Comment by ChrisColeAttensi** [comment]
Doesn't look like its there

![image](https://user-images.githubusercontent.com/86221754/145823292-70650727-539b-445b-888f-85cd07dde4b4.png)

**Comment by bitsandfoxes** [comment]
Thanks for the confirmation. So it's not even copying the framework. I'll take a look and get back to you.

**Comment by fahall** [comment]
We're running into the same problem in our CI/CD pipeline after upgrading from the old sentry unity lite package to the new package on v0.8.0. 

We're running on a Github Actions Mac-OS runner using: https://github.com/kuler90/build-unity , so I suspect this isn't unique to building on Windows.

**Comment by bruno-garcia** [comment]
I believe this was resolved with this PR: https://github.com/getsentry/sentry-unity/pull/457

**Comment by fahall** [comment]
I tried building/deploying today with version 0.9.1 with the same result. I don't think the issue is resolved.

**Comment by bitsandfoxes** [comment]
Could you provide us with the build logs? Copying the framework is the very first thing it does.
![Screenshot 2021-12-21 at 11 28 39](https://user-images.githubusercontent.com/25725783/146914793-0264950c-7188-471d-80db-b704a47f0113.png)

**Comment by fahall** [comment]
[logs_11136.zip](https://github.com/getsentry/sentry-unity/files/7758838/logs_11136.zip)

Logs attached. 

The `Build` action succeeds. It fails during the `Deploy` phase, which includes all the XCode/Testflight steps.

**Comment by fahall** [comment]
Any updates on this?

**Comment by ChrisColeAttensi** [comment]
Yes please. This is a showstopper for us and I've had to downgrade all the way to the old Sentry-Unity package.

**Comment by fahall** [comment]
Bumping again. This issue makes the new SDK unusable. I see PR #496 is mostly a fix to help diagnose the issue and was approved 2 weeks ago but not merged.

Until the issue is resolved, we are also using the old `sentry-unity-lite` package.

**Comment by lucas-zimerman** [comment]
Hi and thanks for the feedback.
The PR #504 should fix the issue. You can also test locally by doing the same change on Sentry.Unity.Editor.iOS.dll.meta

**Comment by ChrisColeAttensi** [comment]
Hi, version 0.9.2 seems to have fixed the issue for me :)

**Comment by bitsandfoxes** [comment]
Apologies for the delay, that one is on me. Thanks @lucas-zimerman for digging through this issue.
For a bit of context: We used to limit the import of the Editor.iOS.dll to OSX only as to not require Unity's iOS module on non-OSX platforms. The reasoning was that iOS builds would only happen on OSX anyway. Unfortunately, it slipped my mind that by placing the `NativeBridge.m` inside Plugins/iOS it would get automatically copied during the build process without the Sentry SDK being set up properly, leading to your issue. 
From our tests (with 2019.4.33f1 and newer), it seems that this limitation is no longer in place so we removed the import restriction.
